docs/input/docs/reference/configuration.md

```diff
@@ -28,21 +28,21 @@ GitHubFlow and GitFlow, probably with others too.
 The `develop` branch is set to `ContinuousDeployment` mode by default as we have
 found that is generally what is needed when using GitFlow.
 
-You can run `GitVersion /showConfig` to see the effective configuration
-(defaults + overrides).
+To see the effective configuration (defaults and overrides), you can run
+`gitversion /showConfig`.
 
-To create your config file just type `GitVersion init` in your repo directory
-after installing via chocolatey and a minimal `GitVersion.yml` configuration
-file will be created. Modify this as you need.
+To create your config file just type `gitversion init` in your repo directory
+after installing via Chocolatey and a minimal `GitVersion.yml` configuration
```

Comment:
Make `installing` a reference link to `https://gitversion.net/docs/usage/cli/installation` so we don't have to mention Chocolatey explicitly here.

```suggestion
after [installing] and a minimal `GitVersion.yml` configuration
```